go.mod

```diff
@@ -6,8 +6,10 @@ require (
 	github.com/ChainSafe/log15 v0.0.0-20180818164646-67afb5ed74ec
```

Comment:
Now that we've released a v1.0.0 for log15, go mod should pick up the actual version. Something like the following:

`require github.com/ChainSafe/log15 v1.0.0`

@dziabko can you update your PR so that v1.0.0 version is referenced? I'm assuming it will happen automatically when go mod looks for the latest packages.

Reply:
`go get -u github.com/ChainSafe/log15` should do the trick